src/components/DataTable/DataTable.tsx

```diff
@@ -32,6 +32,8 @@ export interface DataTableProps {
   headings: string[];
   /** List of numeric column totals, highlighted in the table’s header below column headings. Use empty strings as placeholders for columns with no total. */
   totals?: TableData[];
+  /** Placement of totals row within table, true for t */
+  footerTotals?: boolean;
```

Comment:
Can you spend a bit of time looking through the `polaris-react` codebase for any other examples of conditional-layout-related props similar to this? I'd like to understand what else exists so we can try to align naming conventions.

Reply:
Looking at the `alignFlushLeft` (verb: 'align') prop on `ContextualSaveBar` and the `showHeader` (verb: 'show') prop on `ResourceList`, I think it makes more sense to use a verb-based prop for this.

Reply:
I like this idea, makes the prop seem more action oriented just by reading it